uni/lib/controller/local_storage/app_restaurant_database.dart

```diff
@@ -111,3 +110,27 @@ class RestaurantDatabase extends AppDatabase {
     await txn.delete('restaurants');
   }
 }
+
+List<Restaurant> hideMeals(List<Restaurant> restaurants) {
+  final List<Restaurant> restaurantsCopy = List.from(restaurants);
+  // Hide past and next weeks' meals
```

Comment:
Say it is Tuesday. Shouldn't I be able to see Monday meals?

Reply:
I asked the project coordinator and he mandated that the behaviour of this implementation should replicate the behaviour of sigarra endpoints. Sigarra endpoints only show the current or next days' meals.

Reply:
I asked him again. He kept his stand.